easybuild/easyconfigs/p/PCL/PCL-1.7.2-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'PCL'
+version = '1.7.2'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://pointclouds.org/'
+description = """The Point Cloud Library (PCL) is a standalone, large scale, open project for 2D/3D image and
+ point cloud processing."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/PointCloudLibrary/pcl/archive/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('CMake', '3.5.2')]
+dependencies = [
+    ('Boost', '1.60.0', versionsuffix),
+    ('Eigen', '3.2.8'),
```

Comment:
I don't think that's true...

What if the PCL include files have includes on Eigen?

Although I guess in that case both PCL and Eigen should be listed as build deps.

Not sure what the best course is here, and including it as a real dep is low impact?